cluster/gce/config-default.sh

```diff
@@ -140,6 +140,9 @@ export AGGREGATOR_MASTER_NAME="${INSTANCE_PREFIX}-aggregator"
 export INITIAL_ETCD_CLUSTER="${MASTER_NAME}"
 export MASTER_TAG="${INSTANCE_PREFIX}-master"
 export NODE_TAG="${INSTANCE_PREFIX}-minion"
+# ETCD_LISTEN_CLIENT_IP decides whether etcd servers should also listen to host IP,
```

Comment:
Unless I'm misunderstanding the `configure-kubeapiserver.sh` change, should we also document that, if this env is set, that the kube-apiserver will connect to etcd on the host IP?

Reply:
Good catch. Updated documentation.